website/docs/r/event_orchestration_unrouted.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_event_orchestration_unrouted"
+sidebar_current: "docs-pagerduty-resource-event-orchestration-unrouted"
+description: |-
+  Creates and manages an Unrouted Orchestration for a Global Event Orchestration in PagerDuty.
+---
+
+# pagerduty_event_orchestration_unrouted
+
+An Unrouted Orchestration allows users to create a set of Event Rules that will be evaluated against all events that don't match any rules in the Orchestration's Router.
+
+The Unrouted Orchestration evaluates events sent to it against each of its rules, beginning with the rules in the "start" set. When a matching rule is found, it can modify and enhance the event and can route the event to another set of rules within this Unrouted Orchestration for further processing.
+
+## Example of configuring a Unrouted Rules for an Orchestration
+
+  In this example the user has defined the unrouted orchestration with two rules, each routing to a different service.This example assumes services used in the route_to configuration already exists. So it does not show creation of service resource.As there is a single set, route_to is not defined for the rules.
+
```

Comment:
**Blocker:** "two rules, each routing to a different service" I don't think it's possible to route to a service from the Unrouted Orchestration. Honestly, I think this whole sentence should go away.

```suggestion
```